packages/hooks/src/useWalletsList.ts

```diff
@@ -62,14 +62,14 @@ const currency_to_icon_mapper: Record<string, Record<'light' | 'dark', string>>
 /** A custom hook to get the list of wallets for the current user. */
 const useWalletsList = () => {
     const { client, ui } = useStore();
-    const { loginid, is_authorize } = client;
+    const { loginid } = client;
```

Comment:
Can remove this since this hook is exposed to useAuthorize. Can use authorize_data.loginid instead of deez